Contributing.md

```diff
@@ -56,6 +56,70 @@ discuss pending contributions in order to find a resolution. It is expected that
 small minority of issues be brought to the TC for resolution and that discussion and
 compromise among committers be the default resolution mechanism.
 
+# Triaging
+
+Open issues in this repo are subjected to the triage criteria:
+
+`needs triage`: Issues that are opened and un-attended / unclassified belong to this 
+category, and are either automatically or manually labelled with `needs triage`
+
+`question`: Issues that are not leading upto a bug or an enhancement to any of the 
+express repos, but emanating from the usage of it are classified as `question`
+
+`bug`: Issues that, after investigation, provides a reasonable conviction about
+a potential bug in the express code, are classified as `bug`
+
+`enhancement`: Issues that, after discussion, investigation, and technical assessment,
+found to be a reasonable candidate for a good to have feature, are tagged as `enhancement`.
+
+Raisers may appeal to change the tag with sufficient reasoning.
+Maintainers can change the tag based on the change in inference in the course of investigation.
+
+In all cases, issues may be closed by maintainers if they don't receive timely response when
+further information is sought, or when additional questions are asked.
+
+## Approaches and Best Practices for getting into triage contributions
+
+ - review the organization's [Dashboard][]
+ - pay special attention to these columns: stars, watchers, open issues and contributors
+ - this gives you a general idea about the criticality and health of the repository
+ - pick choose the top n based on that criteria
+ - augment that info with your skills in the middleware space - existing or aspiring
+ - refine the priority based on that
+ - for a specific repo, do the following:
+   - review the project's contribution guideline if present. In a nutshell, pledge yourself to the community's standards and values.
+   - review the documentation. for most of the projects, it is a single page doc
+   - understand the key functions: APIs, their semantics, configurations and use cases
+   - write `own` test programs to re-affirm your understanding on these functions
```

Comment:
suggestion for better readability -  
write your own test programs to re-affirm your understanding of the key functions

Reply:
done